components/sup/src/event.rs

```diff
@@ -378,6 +389,10 @@ mod tests {
         assert_eq!(event.stdout, None);
         assert_eq!(event.stderr, None);
 
+        let interval_in_seconds: u64 = HealthCheckInterval::default().into();
```

Comment:
If I do not specify the type here, the compiler throws a type annotations error and suggests giving the "interval in seconds" variable a type, so I have done so.

Reply:
Changing the type of the interval field in the message could also affect this.